generate_doc.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+handsdown
+mv docs/README.md .
```

Comment:
This moves a generated README.md file to the root folder, essentially overwriting the existing README file. This is not desired.